svgdigitizer/svgdigitizer.py

```diff
@@ -157,29 +149,48 @@ def get_real_values(self, xpathdata):
         ynorm = self.trafo['y'](xpathdata[:, 1])
         return np.array([xnorm, ynorm])
 
+    @cached_property
+    def labeled_paths(self):
+        r"""
+        Return all paths which are grouped/label with text elements, warn if
+        unlabeled or ambiguous paths exist.
+        """
+        labeled_paths = {}
+        texts = self.doc.getElementsByTagName('text')
+        for text in texts:
+
+            group = text.parentNode
+            paths = group.getElementsByTagName('path')
+            # warn for ambiguous paths
+            # gives false positive for scale bars
+            #if len(paths) > 1:
+            #    print("""Multiple paths grouped with single text!
+#Only first will be taken into account. Please review supplied svg file!""")
```

Comment:
We should do something about this so this also works with scalebars.
Can we include the actual text in the message? I think that makes debugging much easier.

Reply:
One "easy" way would be to switch back to a single path per scale bar.